### PR TITLE
[TOOLS-308] speed-up assign to me

### PR DESCRIPTION
**Pull request by herbert-venancio**

### CODE REVIEW

File: src/main/java/objective/taskboard/controller/IssueController.java [hunk 1]
```diff
@@ -107,7 +107,7 @@
 
     @RequestMapping(path = "assign", method = RequestMethod.POST)
     public Issue assign(@RequestBody Issue issue) throws JSONException {
```
Comment (jhonypg) [inline src/main/java/objective/taskboard/controller/IssueController.java]: You can pass just the issueKey on RequestBody.

File: src/main/resources/application-dev.properties [hunk 2]
```diff
@@ -54,7 +54,7 @@ flyway.baselineVersionAsString=1
 # main tshirt custom field
 jira.customfield.t-shirt-size.mainTShirtSizeFieldId=customfield_11457
 
-# Resolvido, Fechado, Cancelado, Or�ado, Done, Recebido
+# Resolvido, Fechado, Cancelado, Or�ado, Done, Recebido
```
Comment (jhonypg) [inline src/main/resources/application-dev.properties]: "Or�ado" has some encoding problem.

File: src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java [hunk 3]
```diff
@@ -74,6 +75,9 @@
     private JiraService jiraBean;
 
     @Autowired
```
Comment (taksan) [inline src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java]: Why another jira client?

Comment (herbert-venancio) [reply src/main/java/objective/taskboard/issueBuffer/IssueBufferService.java]: Mainly to separate old jira-api version from retrofit version. I'll move it to JiraService.

File: src/main/resources/static/elements/taskboard/issue-detail.html [hunk 4]
```diff
@@ -826,18 +828,33 @@
                             'Accept': 'application/json',
                             'Content-Type': 'application/json'
                         },
-                        data: JSON.stringify(this.item),
+                        data: requestBody,
                         success: function (issue) {
```
Comment (jhonypg) [inline src/main/resources/static/elements/taskboard/issue-detail.html]: Change the name "issue" returned from success to a better understanding.

Comment (jhonypg) [inline src/main/resources/static/elements/taskboard/issue-detail.html]: Change the name "issue" returned from success to a better understanding.

**Pull request merged**